app/graphql/types/rock_and_pebble_type.rb

```diff
@@ -4,5 +4,17 @@ class RockAndPebbleType < Types::BaseObject
         field :rock, Types::UserType, null: false
         field :pebble, Types::UserType, null: false
         field :active, Boolean, null: false
+        field :my_rocks, [Types::UserType], null: true
+        field :my_pebbles, [Types::UserType], null: true
+
+        def my_rocks
+          user = User.find(object.pebble_id)
+          user.rocks
+        end
+
+        def my_pebbles
+          user = User.find(object.rock_id)
+          user.pebbles
+        end
```

Comment:
If the fields were renamed as `rocks` and `pebbles`, would it work without having to define resolver methods since those already exist via their ActiveRecord relationships?

Reply:
Hmmm...I can play around with it some more but my first thought is you are going to need a resolver method. The RockAndPebble table only holds a rock_id and a pebble_id so my understanding is that you will need to look up the user first to access the pebble or rock relationship. I could be wrong though.

Reply:
Ohhh I see. No you're right, I was thinking of it as if it were already a `User` object. Thanks!